src/GraphQL/Types/Schema.cs

```diff
@@ -66,11 +75,11 @@ public void Initialize()
 
         public IObjectGraphType Subscription { get; set; }
 
-        public IServiceProvider Services { get; set; }
-
         /// <summary>
-        /// Provides the ability to filter the schema upon introspection to hide types.
+        /// The service provider used to create instances of graph types during initialization of the schema
```

Comment:
Not only graph types. See all usages - directive visitors, middlewares, etc.

Reply:
Updated to just say 'create objects during initialization' and similar.

Reply:
Well, not only during initialization. It is important.